### PR TITLE
Luis #2

### PR DESCRIPTION
**Pull request by gluix20**
Favor probar el test de utils/http

### CODE REVIEW

File: ios/Yester/Info.plist [hunk 1]
```diff
@@ -52,5 +51,22 @@
 			</dict>
 		</dict>
 	</dict>
+	<key>UIAppFonts</key>
```
Comment (albizures) [inline ios/Yester/Info.plist:54]: I think this config is not needed any more right?

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { TouchableHighlight, ViewPropTypes, Text, Dimensions } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+const gradientColors = ['#4444B2', '#37D0D0']
```
Comment (albizures) [inline src/components/Button.js]: there is a util module, called colors,  you can place these colors there

File: src/components/Button.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+const Button = (props) => {
+  return (
+    <TouchableHighlight {...props} elevation={10} style={[props.style, styles.button]}>
+      <LinearGradient start={{x: 0.0, y: 1.0}} end={{x: 1.0, y: 1.0}} style={styles.gradient} colors={gradientColors}>
+        <Text style={styles.text}>{props.title}</Text>
```
Comment (albizures) [inline src/components/Button.js]: we need to localize this text too, so I think you should be using Translate component here instead

File: src/screens/Home/index.js [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+      <FlatList
+        data={this.state.ages}
+        renderItem={this.renderAgeItem}
+        keyExtractor={this.keyExtractor}
+      />
+    </View>
+  )
+
+  getTopicsItems = () => (
+    <View>
```
Comment (albizures) [inline src/screens/Home/index.js]: same here, I think there is no need for this `View`

File: src/screens/Home/index.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+    </View>
+
+  )
+
+  keyExtractor = (item, index) => {
+    return index.toString()
+  }
+
+  getAgesItems = () => (
+    <View>
```
Comment (albizures) [inline src/screens/Home/index.js]: I think there is no need for this `View`

File: src/screens/Home/index.js [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+        {this.getAgesItems()}
+      </Container>
+    )
+  }
+}
+
+const localStyles = StyleSheet.create({
+  container: {
+    flex: 1,
```
Comment (albizures) [inline src/screens/Home/index.js]: `Container` already has `flex` as 1, so I think there is no need for adding it here again

File: src/screens/Home/index.js [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+    paddingTop: 22,
```
Comment (albizures) [inline src/screens/Home/index.js]: are you sure you need this margin? `Container`already has a `marginTop`

File: src/screens/CreateAccount.js [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    this.props.navigation.navigate('Facebook')
+  }
+
+  onPressSignIn = () => {
+    this.props.navigation.navigate('SignIn')
+  }
+
+  render () {
+    return (
+      <ScrollView style={[localStyles.container]} >
```
Comment (albizures) [inline src/screens/CreateAccount.js]: keep in mind that we should be using `Container` on every screen, no need to do it in this PR though.

**Review by albizures — CHANGES_REQUESTED** [review]

File: ios/Yester/Info.plist (hunk 1 above)
Comment (gluix20) [reply ios/Yester/Info.plist:54]: I don't think so. I uninstalled already both Icons packages, but can't find how to deinstall this UIAppFonts.

File: src/components/Button.js (hunk 2 above)
Comment (gluix20) [reply src/components/Button.js]: Agree. Done.

File: src/components/Button.js (hunk 3 above)
Comment (gluix20) [reply src/components/Button.js]: Agree. Done.

File: src/screens/Home/index.js (hunk 6 above)
Comment (gluix20) [reply src/screens/Home/index.js]: I removed that style, to use only the Container component default style.

File: src/screens/Home/index.js (hunk 7 above)
Comment (gluix20) [reply src/screens/Home/index.js]: I removed that style, to use only the Container component default style.

**Pull request closed without merging**